### PR TITLE
Services - namespaces

### PR DESCRIPTION
**Pull request by laics1984**
- UX
- Load existing namespaces
- Multisig accounts balance validation

### CODE REVIEW

File: src/util/namespacesUtils.ts [hunk 1]
```diff
@@ -0,0 +1,123 @@
...
+    return registerRootNamespaceTransaction.maxFee.compact();
+  }
+
+  static getSubNamespaceTransactionFee =(networkType: NetworkType, generationHash: string, namespaceName: string, subNamespace: string) :number => {
+    let buildTransactions = new BuildTransactions(networkType, generationHash);
+    const registerSubNamespaceTransaction = buildTransactions.registersubNamespace(namespaceName, subNamespace);
+    return registerSubNamespaceTransaction.maxFee.compact();
+  }
+
+  static calculateDuration = (duration: number): number =>{
```
Comment (sleepyOwl14) [inline src/util/namespacesUtils.ts:33]: maybe the param name be more specific ? durationInDay

File: src/util/namespacesUtils.ts [hunk 2]
```diff
@@ -0,0 +1,123 @@
...
+        if (a.level < b.level) return -1;
+        return 0;
+      });
+    }
+    return namespacesArr;
+  }
+
+  static listRootNamespaces = (address:string) => {
+    const namespacesAccount = walletState.currentLoggedInWallet.accounts.find((account) => account.address === address).namespaces.filter(namespace => namespace.active === true);
+    const namespacesList = namespacesAccount.length;
```
Comment (sleepyOwl14) [inline src/util/namespacesUtils.ts:75]: just the variable not accurate and confusing, 
namespacesAccount -> accountNamespaces
namespaceList -> namespacesNum

File: src/util/namespacesUtils.ts [hunk 3]
```diff
@@ -0,0 +1,123 @@
...
+    let namespacesArr = [];
+    if(namespacesList > 0){
+      namespacesAccount.forEach((namespaceElement) => {
+        const level = namespaceElement.name.split('.');
```
Comment (sleepyOwl14) [inline src/util/namespacesUtils.ts:79]: this is not accurate, the name does not have the full name, should check for parentId

**Review by sleepyOwl14 — CHANGES_REQUESTED** [review]
issues highlighted

File: src/util/namespacesUtils.ts (hunk 1 above)
Comment (laics1984) [reply src/util/namespacesUtils.ts:33]: Sure, noted

File: src/util/namespacesUtils.ts (hunk 2 above)
Comment (laics1984) [reply src/util/namespacesUtils.ts:75]: Okay

File: src/util/namespacesUtils.ts (hunk 3 above)
Comment (laics1984) [reply src/util/namespacesUtils.ts]: Updated with trace by parentId. Thanks

Comment (sleepyOwl14) [reply src/util/namespacesUtils.ts]: ok, you can use split by `.`. The stored name actually is the full name

Comment (laics1984) [reply src/util/namespacesUtils.ts:79]: Have reverted back to the previous. Thanks

**Review by sleepyOwl14 — APPROVED** [review]

**Pull request merged**